extendr-api/src/wrapper/matrix.rs

```diff
@@ -254,72 +255,121 @@ where
     }
 }
 
-impl<'a, T: 'a> TryFrom<Robj> for RColumn<T>
+impl<T> TryFrom<&Robj> for RColumn<T>
```

Comment:
Note that the LHS is the impl for`Robj` while this is `&Robj`. Below is the handling for `Robj`.